### ISSUE TITLE
Change 'Supported versions'

### ISSUE THREAD

**Issue by tasty0tomato** [phpmyadmin/phpmyadmin#17430]
### Describe the bug

Maybe not a bug. Just something about the website. This may not be the right place to report this.
https://www.phpmyadmin.net/downloads/
The **Supported versions** part says that Version 5.0's **End of regular support** is still TBD. But according to https://www.phpmyadmin.net/news/2020/10/15/phpmyadmin-497-and-504-are-released/, version 5.0 is not supported.

### Expected behavior

Version 5.0's **End of regular support** is 2020-10-15

### Screenshots

![image](https://user-images.githubusercontent.com/48764449/157175615-e5783e62-c392-4550-a593-a8182da5088c.png)
![image](https://user-images.githubusercontent.com/48764449/157175736-e2be8fee-39ad-4f2a-a471-e5dfc5b73028.png)

**Comment by ibennetch** [comment]
You are quite correct. 5.0 has been replaced by 5.1 and we should update this to reflect that. Thanks for pointing that out.

Assigning this to myself.

**Comment by ibennetch** [comment]
Thanks for reporting this, I have published an update to correct this which you should see reflected on the site now.
